### PR TITLE
[Spaces] Small note about `pre-requirements.txt`

### PR DESCRIPTION
**Pull request by cbensimon**

### CODE REVIEW

File: docs/hub/spaces-dependencies.md
Comment (julien-c) [inline docs/hub/spaces-dependencies.md:21]:
```suggestion
If you need other Python packages to run your app, add them to a **requirements.txt** file at the root of the repository. The Spaces runtime engine will create a custom environment on-the-fly (you can also add a **pre-requirements.txt** file describing dependencies that will be installed before your main dependencies. It can be useful if you need to update pip itself).
```

**Review by julien-c — APPROVED** [review]
cool feature!

cc @osanseviero @abidlabs too fyi (i didn't know about this)

File: docs/hub/spaces-dependencies.md
Comment (osanseviero) [inline docs/hub/spaces-dependencies.md:21]:
```suggestion
If you need other Python packages to run your app, add them to a **requirements.txt** file at the root of the repository. The Spaces runtime engine will create a custom environment on-the-fly. You can also add a **pre-requirements.txt** file describing dependencies that will be installed before your main dependencies. It can be useful if you need to update pip itself.
```

**Review by osanseviero — APPROVED** [review]
LGTM to merge once the internal PR is merged :fire: thanks!

**Comment by christophe-rannou** [comment]
`pre-requirements.txt` feature is merged and live

**Pull request merged**
